decidim-surveys/spec/system/admin_manages_surveys_spec.rb

```diff
@@ -11,7 +11,7 @@
            published_at: nil)
   end
   let!(:questionnaire) { create(:questionnaire) }
-  let!(:survey) { create(:survey, component:, questionnaire:) }
+  let!(:survey) { create(:survey, component:, published_at: Time.current, questionnaire:) }
```

Comment:
```suggestion
  let!(:survey) { create(:survey, :published, component:, questionnaire:) }
```
Just use the new factory trait

Reply:
Done